serverless.ts

```diff
@@ -274,6 +274,22 @@ const serverlessConfig: AWS = {
       IsDev: {
         'Fn::Equals': ['${self:provider.stage}', 'dev'],
       },
+      IsProd: {
```

Comment:
what do you need `isProd` for?

Reply:
We just want to create a bucket for `prod` or `dev`.  This condition is used on the `IsDevOrProd` condition.

Reply:
oh i see, yeah, i guess that works too